lib/pusher-pub-sub-gateway.js

```diff
@@ -65,8 +65,12 @@ class PusherPubSubGateway {
   }
 }
 
-function createDisconnectedPusherClient (key) {
-  const client = new Pusher(key, {encrypted: true}) // automatically connects to pusher
+function createDisconnectedPusherClient (key, options) {
+  const connectOptions = Object.assign({
+    encrypted: true,
+  }, options || {})
```

Comment:
Do you need the `|| {}` with `Object.assign`?

Reply:
Not necessary.